the-badass-todo-list-part-1/list.js

```diff
@@ -29,11 +30,14 @@
     document.querySelector('#todo-list-wrapper').appendChild(list);
   }
 
+  // Quiz #2
```

Comment:
* `saveData` should be used in a more complete version.

* Good point. Usually we will have some `constants` to map things like this.